src/test/java/uk/ac/ebi/subs/apisupport/ApiSupportServiceMarkSubmittedTest.java

```diff
@@ -37,13 +38,13 @@
     public void markDraftAsSubmitted(){
         apiSupportService.markContentsAsSubmitted(submission);
 
-        ProcessingStatus expectSubmitted = processingStatusRepository.findById(draftSample.getProcessingStatus().getId())
+        ProcessingStatus expectSubmitted = Optional.of(processingStatusRepository.findOne(draftSample.getProcessingStatus().getId()))
                 .orElse(null);
```

Comment:
isnt it better to just not use Optional.of() in such places? it might confuse a new developers and will set a bad example for them. i also dont see spending cpu time on such needless thing, no matter how small.

Reply:
It is a test, but if you want I can remove `Optional` from here